### ISSUE TITLE
New-HPOVStorageVolumeTemplate will not create Thin Dedupe templates

### ISSUE THREAD

**Issue by daveolker** [HewlettPackard/POSH-HPOneView#283]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
New-HPOVStorageVolumeTemplate -Capacity 100 -Name SVT-Demo-Shared-TPDD12 -StoragePool CPG-SSD -Shared -SnapshotStoragePool CPG-SSD -StorageSystem ThreePAR-2 -ProvisionType TPDD

This command should create a template with ProvisionType set to TPDD.  The same command works fine with the "Thin" and "Full" provision types.

### Actual Behavior
```Powershell
New-HPOVStorageVolumeTemplate : The provided value in the property 'provisioningType' is not supported by the enum attribute. Provide a value in the 
'default' attribute that is also contained in the 'enum' attribute and retry the request. 
At line:1 char:1
+ New-HPOVStorageVolumeTemplate -Capacity 100 -Name SVT-Demo-Shared-TPD ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Send-HPOVRequest:String) [New-HPOVStorageVolumeTemplate], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperation,New-HPOVStorageVolumeTemplate
```
### Steps to reproduce

See above.

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`):

PowerShellLibraryVersion : 3.10.1377.2109
LibraryPath : C:\Program Files\WindowsPowerShell\Modules\HPOneView.310

HPE OneView Appliance Version: 3.10.05-0302465

Output from $PSVersionTable on your Windows Host:

Name Value

PSVersion 5.1.14393.1480
PSEdition Desktop
PSCompatibleVersions {1.0, 2.0, 3.0, 4.0...}
BuildVersion 10.0.14393.1480
CLRVersion 4.0.30319.42000
WSManStackVersion 3.0
PSRemotingProtocolVersion 2.3
SerializationVersion 1.1.0.1

**Comment by ChrisLynchHPE** [comment]
Thanks for reporting this.  I found and fixed the issue.  Will be addressed in the next update.

**Comment by ChrisLynchHPE** [comment]
I have addressed this issue with in [Release 3.10.1416.3042](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.10.1416.3042).  Please confirm, and we can then close this issue.

**Comment by daveolker** [comment]
Thanks Chris.  I just tested the fix and it works perfectly.
